pkg/apis/batch/v1alpha1/job.go

```diff
@@ -317,6 +317,20 @@ type JobStatus struct {
 	// The resources that controlled by this job, e.g. Service, ConfigMap
 	// +optional
 	ControlledResources map[string]string `json:"controlledResources,omitempty" protobuf:"bytes,11,opt,name=controlledResources"`
+
+	// The time when the job starts running
+	// +optional
+	StartTime *metav1.Time `json:"startTime,omitempty" protobuf:"bytes,12,opt,name=startTime"`
+
+	// The time when the job phase turn to completed or failed
+	// +optional
+	EndTime *metav1.Time `json:"endTime,omitempty" protobuf:"bytes,13,opt,name=endTime"`
+
+	// Which conditions caused the current job state.
+	// +optional
+	// +patchMergeKey=status
+	// +patchStrategy=merge
+	Conditions []JobCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"status" protobuf:"bytes,14,rep,name=conditions"`
```

Comment:
If we have `Conditions`, startTime & endTime are not necessary :)

Reply:
ok, I've already deleted it. 
Is there anything else we need to add in `conditions`